gtk4/src/dialog.rs

```diff
@@ -1,6 +1,7 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
-use crate::{Dialog, DialogExt, DialogFlags, ResponseType, Widget, WidgetExt, Window};
```

Comment:
Actually it would be better to `use crate::prelude::*` here everywhere. That way all this manual code does not have to be updated if one of the functions in question is moved from an `Ext` to an `ExtManual` trait or the other way around.